pre-commit.Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Developer:
+# ---------
+# Name:      Maik Ellerbrock
+#
+# GitHub:    https://github.com/ellerbrock
+# Twitter:   https://twitter.com/frapsoft
+# Docker:    https://hub.docker.com/u/ellerbrock
+# Quay:      https://quay.io/user/ellerbrock
+#
+# Description:
+# -----------
+# Bash Shell v.5 bats and deps for development
```

Comment:
as I said, lets just use the minimal dockerfile that you wrote for the pre-commit- no need for complicated stuff